ivy/functional/ivy/data_type.py

```diff
@@ -631,21 +631,23 @@ def unset_default_dtype():
 
 
 # noinspection PyShadowingNames
-
-
-def set_default_int_dtype(int_dtype: Union[ivy.Dtype, str]):
-    """Summary.
+def set_default_int_dtype(int_dtype: Union[ivy.Dtype, str]) -> None:
+    """Default the type to int and append the type to stack.
 
     Parameters
     ----------
     int_dtype
+        int data type
+
+    Returns
+    -------
+        Return None
 
```

Comment:
Please add some examples to this function, you can refer to the attached link for docstrings examples formatting as the parameters of this method is related to DType instead of arrays
https://lets-unify.ai/ivy/deep_dive/11_docstring_examples.html#docstring-examples 

Once an int_dtype is set, you can get the current dtype by using `ivy.default_int_dtype()`

Reply:
Hi! I have made a new commit!

Reply:
Hello! I'll review them soon, in one or two days! Thanks!